packages/opentelemetry-plugin-express/README.md

```diff
@@ -45,6 +45,10 @@ const provider = new NodeTracerProvider();
 
 See [examples/express](https://github.com/open-telemetry/opentelemetry-js/tree/master/examples/express) for a short example.
 
+### Caveats
+
+Because of the way express works, it's hard to correctly compute the time took by asynchronous code to complete. For this reason the time you'll see reported for asynchronous middlewares and request handlers will **not** represent the time it really took.
```

Comment:
```suggestion
Because of the way express works, it's hard to correctly compute the time taken by asynchronous middlewares and request handlers. For this reason, the time you'll see reported for asynchronous middlewares and request handlers will only represent the synchronous execution time, and **not** any asynchronous work.
```